src/test/java/utils/seleniumWrapper.java

```diff
@@ -0,0 +1,35 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+
+public class seleniumWrapper {
+
+    WebDriver driver = new EdgeDriver();
+
+       public boolean clickElement(By locator){
+        driver.findElement(locator).click();
+        return true;
+
+    }
+
+    //function 2 - isDisplayed
+    public boolean displayElement(By locator){
+        driver.findElement(locator).isDisplayed();
```

Comment:
driver.findElement(locator).isDisplayed() returns boolean result as true if element is displayed and false if it is not, so you can simply return the value returned by this function

Reply:
Made the changes.